src/components/ContentSidebar/SidebarVersions.js

```diff
@@ -7,23 +7,26 @@
 import React from 'react';
 import VersionHistoryLink from 'box-react-ui/lib/features/item-details/VersionHistoryLink';
 import withErrorHandling from './withErrorHandling';
-import type { FileVersions } from '../../flowTypes';
+import type { BoxItem, FileVersions } from '../../flowTypes';
+import { BOX_NOTE_EXTENSION } from '../../constants';
 
 type Props = {
     onVersionHistoryClick?: Function,
-    versions?: FileVersions
+    versions?: FileVersions,
+    file: BoxItem
 };
 
 const SidebarVersions = ({
     onVersionHistoryClick,
     versions = {
         total_count: 0,
         entries: []
-    }
+    },
+    file
 }: Props) => {
     const { total_count } = versions;
 
-    if (!total_count) {
+    if (!total_count || file.extension === BOX_NOTE_EXTENSION) {
```

Comment:
I would create a simple utility for isBoxNote, maybe inside util/file.js?
So that we just use that than repeat ===

Or you can name the utility isExcludedFileType or something